src/components/RuleCard/index.jsx

```diff
@@ -740,9 +742,13 @@ function RuleCard({
           </Button>
           {requiresSignoff &&
             (user && user.email in rule.scheduledChange.signoffs ? (
-              <Button color="secondary">Revoke Signoff</Button>
+              <Button color="secondary" onClick={onRevoke}>
+                Revoke Signoff
+              </Button>
             ) : (
-              <Button color="secondary">Signoff as</Button>
+              <Button color="secondary" onClick={onSignoff}>
+                Signoff
+              </Button>
```

Comment:
We should probably hide these buttons when `readOnly` is true.

Reply:
This was already the case actually (via https://github.com/mozilla-frontend-infra/balrog-ui/pull/116/files/d065ff5fa07f12dc4aae77f2f60f6a8a4f8241ce#diff-a0795e915062262d1dae44d4335bf63cR719). However, I noticed that we don't set readOnly when there's no logged in user - I've corrected that.